### PR TITLE
Add vivo to karmada adopters

### PR DESCRIPTION
**Pull request by riverzhang**
Signed-off-by: zhangrong <zhangrong@vivo.com>

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/casestudies/adopters.md [hunk 1]
```diff
@@ -21,3 +21,4 @@ All organisations are sorted alphabetically.
 | Trip.com                  | https://www.trip.com/       | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada            | TBD                                                                           |
 | VIPKID                    | https://www.vipkid.com/     | Use Karmada to build multi-cluster platform with heterogeneous resources and clusters   | [Building a PaaS Platform with Karmada to Run Containers --VIPKID](vipkid.md) |
 | Youzan                    | https://www.youzan.com/     | Manage both Stateless and Stateful on-line workloads across multiple clusters           | TBD                                                                           |
+| Vivo                      | https://www.vivo.com/       | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada            | TBD                                                                           |
```
Comment (RainbowMango) [inline docs/casestudies/adopters.md:24]: Given `All organisations are sorted alphabetically.`, I think you can put the new item right after `VIPKID` and before `Youzan`.

Comment (riverzhang) [reply docs/casestudies/adopters.md:23]: done

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

/hold
I see you are using the email with the suffix `@vivo.com`, I guess you haven't put this email to [GitHub Emails], so GitHub can't recognize you and can't count your contributions.

Putting your email to GitHub is not a requirement, just a reminder.

**Comment by riverzhang** [comment]
> /lgtm /approve
> 
> /hold I see you are using the email with the suffix `@vivo.com`, I guess you haven't put this email to [GitHub Emails], so GitHub can't recognize you and can't count your contributions.
> 
> Putting your email to GitHub is not a requirement, just a reminder.

Thanks, I updated my email on GitHub.

**Comment by RainbowMango** [comment]
/hold cancel
Thank you!!!!

**Pull request merged**